packages/addons/service/tvheadend42/source/bin/tvheadend42.start

```diff
@@ -37,6 +37,12 @@ if [ -f "$ADDON_HOME/dvr/config/dvr-config" ]; then
   rm $ADDON_HOME/dvr/config/dvr-config
 fi
 
+# workaround to create default Tvh directories without entering samba share
+if [ ! -d "/storage/recordings" ]; then
+  mkdir -p /storage/recordings
+  mkdir -p /storage/picons/{tvh,vdr}
```

Comment:
This won't work with `busybox` `ash` - it will create:
```
rpi22:~ # ls -la /storage/picons/
total 20
drwxr-xr-x    3 root     root          4096 Nov 18 22:41 .
drwxr-xr-x   68 root     root         12288 Nov 18 22:41 ..
drwxr-xr-x    2 root     root          4096 Nov 18 22:41 {tvh,vdr}
```

Reply:
meh, tx